client/app/views/receiver/tip.html

```diff
@@ -79,19 +79,65 @@
   <div class="row">
     <div class="col-md-6">
       <div class="TipPageSectionTitle" data-translate>Submission</div>
-      <div class="col-md-12">
-        <dl>
-          <dt>
-            <span>{{::tip.context_name}}</span>
+
```

Comment:
all this changes applied to receiver/tip.html should be done to whistleblower/tip.html

be homogenous my friend! :)

i will take care of doing it by finalizing the pull request and i will try also to use partial templates whenever possible.